.github/workflows/doc.yml

```diff
@@ -32,6 +33,38 @@ jobs:
     - uses: actions/checkout@v3
     - name: Register Sphinx problem matcher
       run: echo "::add-matcher::.github/problem-matchers/sphinx.json"
+    - name: 'Set up Python'
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3'
+        cache: 'pip'
+        cache-dependency-path: 'Doc/requirements.txt'
+    - name: 'Install build dependencies'
+      run: make -C Doc/ venv
+    - name: 'Check documentation'
+      run: make -C Doc/ SPHINXOPTS="-q -W --keep-going" check
+    - name: 'Build HTML documentation'
+      run: make -C Doc/ SPHINXOPTS="-q -W --keep-going" html
```

Comment:
The `-W` since to be already included.  The output command shows:
```
PATH=./venv/bin:$PATH sphinx-build -b html -d build/doctrees  -j auto -q -W --keep-going -W . build/html 
```
(Not your fault since you were just copy-pasting, but maybe it can be removed here and above?)

Reply:
`-W` comes from SPHINXERRORHANDLING, I moved `-W` and `--keep-going` to SPHINXERRORHANDLING (If better to just put `-q --keep-going` in SPHINXOPTS I'll do so, I don't know if it would be possible for the `-W` to be lost somehow in other make configuration)

A